secant.xcodeproj/project.pbxproj

```diff
@@ -3546,8 +3546,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/zcash/ZcashLightClientKit.git";
 			requirement = {
-				kind = revision;
-				revision = c88fd684e8bea7f6c8c0ff799220a42361239af9;
+				branch = feature/831_solve_async_nonasync;
```

Comment:
warning: our git workflow will delete this branch once it's merged and will cease to exist.

Reply:
Changes are merged to main and the branch is gone time. It's time to switch to install SDK from the commitl